spec/tcr_spec.rb

```diff
@@ -440,7 +440,7 @@
     }
 
     TCR.use_cassette("test") do
-      sock = Socket.tcp("towel.blinkenlights.nl", 23, {})
+      sock = Socket.tcp("towel.blinkenlights.nl", 23)
       expect(sock).to be_a(TCR::RecordableTCPSocket)
     end
```

Comment:
If this spec uses a cassette named "test"... does that mean a new cassette file should be checked in as well? Is there a chance it didn't record or isn't behaving how we think?

Reply:
Thanks for the follow-up!

My understanding is that the whole spec is written to 1) delete any existing `test.xxx` cassette files before each example, 2) record a new cassette in each example, and then 3) delete the newly recorded cassette after each example (per [this `around` block](https://github.com/robforman/tcr/blob/72fa8427350c2968871afd66c88cfc094b068743/spec/tcr_spec.rb#L17-L25))

In either case, the success or failure of the cassette recording isn't quite the focus here: rather, the change here is to ensure that `Socket.tcp` does not strictly **require** three arguments (because Ruby's STDLIB `Socket.tcp` only requires two).

Before this PR, removing the `{}` would have caused the test to fail (`ArgumentError`); with this PR, you can remove it, and the test will still pass. Thus, the presence of a cassette file (or any problems saving a new recording) make no difference to the fidelity of the test.